homeassistant/components/stream/worker.py

```diff
@@ -562,7 +563,7 @@ def stream_worker(
 
     dts_validator = TimestampValidator(
         int(1 / video_stream.time_base),
-        1 / audio_stream.time_base if audio_stream else 1,
+        int(1 / audio_stream.time_base) if audio_stream else 1,
     )
```

Comment:
The `TimestampValidator` class expects two integers
https://github.com/home-assistant/core/blob/0eea3176d6b6bf871acc7a340f748af88615637e/homeassistant/components/stream/worker.py#L444-L448

`1 / audio_stream.time_base` is the inverse `time_base` and `>1`.